File: Programming_Task_04_ShaliniSengar/Python_Files/Student_Information_Manager.py
# Function to display student information

def display_student(name, roll_no, branch, semester):
    print("\nStudent Information")
    print("-------------------")
    print("Name      :", name)
    print("Roll No   :", roll_no)
    print("Branch    :", branch)
    print("Semester  :", semester)

# Taking input from user
name = input("Enter Student Name: ")
roll_no = input("Enter Roll Number: ")
branch = input("Enter Branch: ")
semester = input("Enter Semester: ")

# Function call
display_student(name, roll_no, branch, semester)

File: Programming_Task_04_ShaliniSengar/C_Files/Student_Information_Manager.c
#include <stdio.h>

// Function to display student information
void displayStudent(char name[], int rollNo, char branch[], int semester)
{
    printf("\nStudent Information\n");
    printf("-------------------\n");
    printf("Name      : %s\n", name);
    printf("Roll No   : %d\n", rollNo);
    printf("Branch    : %s\n", branch);
    printf("Semester  : %d\n", semester);
}

int main()
{
    char name[50], branch[30];
    int rollNo, semester;

    printf("Enter Student Name: ");
    scanf(" %[^\n]", name);

    printf("Enter Roll Number: ");
    scanf("%d", &rollNo);

    printf("Enter Branch: ");
    scanf("%s", branch);

    printf("Enter Semester: ");
    scanf("%d", &semester);

    // Function call
    displayStudent(name, rollNo, branch, semester);

    return 0;
}